Added Anonymous Metrics Send Opt-In Setting

Samyoul opened the pull request:
Added new setting to persist the anonymous metrics sending opt-in

Samyoul committed:
Added anon metrics send opt in setting

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [x] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status_go.html))?

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | dd1efe6a | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2195/1/) | 2021-04-12 14:15:36 | ~2 min | `linux` | [:package:`zip`](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-210412-141252-dd1efe.zip) |
| :heavy_check_mark: | dd1efe6a | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2195/1/) | 2021-04-12 14:17:09 | ~4 min | `ios` | [:package:`zip`](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-210412-141252-dd1efe.zip) |
| :heavy_check_mark: | dd1efe6a | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2195/1/) | 2021-04-12 14:18:07 | ~5 min | `android` | [:package:`aar`](https://status-go.ams3.digitaloceanspaces.com/status-go-android-210412-141252-dd1efe.aar) |
| | | | | | | |
| :heavy_check_mark: | 93b04d9e | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2195/2/) | 2021-04-12 14:40:24 | ~2 min | `linux` | [:package:`zip`](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-210412-143812-93b04d.zip) |
| :heavy_check_mark: | 93b04d9e | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2195/2/) | 2021-04-12 14:40:30 | ~2 min | `ios` | [:package:`zip`](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-210412-143812-93b04d.zip) |
| :heavy_check_mark: | 93b04d9e | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2195/2/) | 2021-04-12 14:42:28 | ~4 min | `android` | [:package:`aar`](https://status-go.ams3.digitaloceanspaces.com/status-go-android-210412-143812-93b04d.aar) |

Samyoul committed:
resolved rebase conflict, renamed migration to use unixtimestamp

Theres always conflicts with migrations using sequential numbers, less so with unix timestamp
